distro_test.go

```diff
@@ -140,16 +146,24 @@ configuration:
     - LANG: en_US.UTF-8
     - PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
     - TERM: xterm-256color
-`, realDistro.Name())},
+`, realDistro.Name(), realID),
+		},
 		"fake distro": {
 			distro: &fakeDistro,
-			wants: fmt.Sprintf(`distro: %s
-configuration: error in GetConfiguration: failed syscall to WslGetDistributionConfiguration
-`, fakeDistro.Name())},
+			wants: fmt.Sprintf(`name: %s
+guid: |
+  distro is not registered
+configuration: |
+  error in GetConfiguration: failed syscall to WslGetDistributionConfiguration
+`, fakeDistro.Name()),
+		},
 		"wrong distro": {
 			distro: &wrongDistro,
-			wants: fmt.Sprintf(`distro: %s
-configuration: error in GetConfiguration: failed to convert %q to UTF16
+			wants: fmt.Sprintf(`name: %s
+guid: |
+  distro is not registered
+configuration: |
+  error in GetConfiguration: failed to convert %q to UTF16
```

Comment:
Note, the wants starts to be long. If it gets any longer, that’s the signal we should use a golden file.

Reply:
Normally I'd resist about touching the filesystem in tests for performance reasons, but our bottleneck (WSL registration) is several orders of magnitudes slower so I supposed that'd be fine.

Reply:
But in general, we can run tests in parallel, so golden files with parallel tests is ok :) I don’t think the additional second is the most annoying part here.

Reply:
Good point about parallel tests. I'm just too used to Python and C++ 😅